### PR TITLE
v0.1: Fix uses of https://istio.io so the site works properly in the archive.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: _docs/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ type: markdown
 # Welcome - Istio 0.1
 
 Welcome to Istio's documentation archive for the 0.1 release.
-Follow [this link for the most current documentation](https://istio.io/docs/).
+Follow [this link for the most current documentation]({{home}}/docs/).
```
Comment (ldemailly) [inline _docs/index.md]: actually that one does want to go to istio.io as it's saying it's an archive

File: _posts/2017-05-24-0.1-announcement.md [hunk 2]
```diff
@@ -8,8 +8,10 @@ layout: post
 type: markdown
 redirect_from: "/blog/0.1-announcement.html"
 ---
+{% include home.html %}
 
-Google, IBM, and Lyft are proud to announce the first public release of [Istio](https://istio.io/): an open source project that provides a uniform way to connect, secure, manage and monitor microservices. Our current release is targeted at the [Kubernetes](https://kubernetes.io/) environment; we intend to add support for other environments such as virtual machines and Cloud Foundry in the coming months.
+Google, IBM, and Lyft are proud to announce the first public release of [Istio]({{home}}): an open source project that provides a uniform way
```
Comment (ldemailly) [inline _posts/2017-05-24-0.1-announcement.md]: I don't think having the blogs going to the main site is bad at all

**Comment by geeknoid** [comment]
PTAL

File: tmp/.htmlproofer/cache.log [hunk 3]
```diff
@@ -0,0 +1 @@
...":["./_site/blog/2018/egress-tcp.html"],"status":200,"message":""},"https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables":{"time":"2018-02-08 07:22:40 -0800","filenames":["./_site/blog/2018/egress-tcp.html"],"status":200,"message":""},"https://raw.githubusercontent.com/istio/istio/master/samples/sleep/sleep.yaml":{"time":"2018-02-08 07:22:40 -0800","filenames":["./_site/docs/setup/kubernetes/sidecar-injection.html"],"status":200,"message":""},"https://raw.githubusercontent.com/istio/istio/master/install/kubernetes/webhook-create-signed-cert.sh":{"time":"2018-02-08 07:22:40 -0800","filenames":["./_site/docs/setup/kubernetes/sidecar-injection.html"],"status":200,"message":""},"https://www.mysql.com":{"time":"2018-02-08 07:22:40 -0800","filenames":["./_site/blog/2018/egress-tcp.html"],"status":200,"message":""},"https://www.mongodb.com":{"time":"2018-02-08 07:22:40 -0800","filenames":["./_site/blog/2018/egress-tcp.html"],"status":200,"message":""},"https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster":{"time":"2018-02-08 07:22:40 -0800","filenames":["./_site/docs/setup/kubernetes/sidecar-injection.html"],"status":200,"message":""}}
```
Comment (ldemailly) [inline tmp/.htmlproofer/cache.log]: remove and add to .gitignore

File: _posts/2017-05-24-0.1-announcement.md [hunk 4]
```diff
@@ -8,8 +8,10 @@ layout: post
 type: markdown
 redirect_from: "/blog/0.1-announcement.html"
 ---
+{% include home.html %}
```
Comment (ldemailly) [inline _posts/2017-05-24-0.1-announcement.md]: out of curiosity what does that do/change ?

**Comment by geeknoid** [comment]
The post contains several instances of {{home}} which without this include
statement resolves to "". This happened to work when the website is at the
root of the domain, but failed within the archive. Actually pulling in the
include defines {{home}} to the right thing and fixes the links.


On Mon, Feb 12, 2018 at 10:24 AM, Laurent Demailly <notifications@github.com

**Comment by ldemailly** [comment]
oic, thanks!

**Pull request merged**